content/en/docs/reference/glossary/static-pod.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Static Pod
+id: static-pod
+date: 2091-02-12
+full_link: /docs/tasks/administer-cluster/static-pod/
+short_description: >
+  A pod configured directly into Kubelet on a specific node
```

Comment:
I don't think "into" is the right preposition here. Probably "by the" makes more sense.

Reply:
If I configure a static pod by editing the appropriate Kubelet configuration file, it's me that's doing the configuring and Kubelet that's being configured.
I want to summarize that relationship.

I'll have a think about wording here.

Reply:
the kubelet (note capitalization! it's lowercase, please change throughout) is an agent that manages pods by means of specs/config files. "into" suggests that the kubelet contains the pod, which it doesn't. The kubelet is configured to manage the pod (although that's the wrong emphasis here, too). Try reusing more of the language from the task topic -- it's pretty solid.